### DISCUSSION TITLE
what is Inconclusive mean ,how to handle it?

### DISCUSSION THREAD

**Discussion by 13567436138** [argoproj/argo-rollouts#1476] [Q&A]
NAME                                   KIND         STATUS          AGE    INFO
⟳ reviews-v1                           Rollout      ॥ Paused        128m   
├──# revision:14                                                           
│  ├──⧉ reviews-v1-9db5f6df9           ReplicaSet   ✔ Healthy       123m   canary
│  │  └──□ reviews-v1-9db5f6df9-wzdvn  Pod          ✔ Running       4m46s  ready:2/2
│  ├──α reviews-v1-9db5f6df9-14-2      AnalysisRun  ? Inconclusive  4m22s  ? 1
│  └──α reviews-v1-9db5f6df9-14-2.1    AnalysisRun  ? Inconclusive  3m48s  ? 1
├──# revision:13                                                           
│  └──⧉ reviews-v1-54b55d79d           ReplicaSet   ✔ Healthy       128m   stable
│     └──□ reviews-v1-54b55d79d-tjgr4  Pod          ✔ Running       82m    ready:2/2
└──# revision:12                                                           
   └──α reviews-v1-9db5f6df9-12-2      AnalysisRun  ⚠ Error         11m    ⚠ 5

**Comment by kostis-codefresh** [comment]
It is explained here https://argoproj.github.io/argo-rollouts/features/analysis/#inconclusive-runs

Basically it is the case where neither your success or failure numbers are hit.

For example if you say that that success is > 90% and failure is <30% then a value of 60% is neither success or failure (and therefore inconclusive).

To fix
- check the thresholds of your analysis
- check what is actually returned by your metrics during the analysis.
